### ISSUE TITLE
Training instructions for MOT 17

### ISSUE THREAD

**Issue by sawhney-medha** [MCG-NJU/MeMOTR#10]
Hello! Can you please upload instructions to use the the training script for MOT17? Thank you!!

**Comment by HELLORPG** [comment]
My bad. I forgot to upload this config file into [configs](https://github.com/MCG-NJU/MeMOTR/tree/main/configs) but only put it [here](https://github.com/MCG-NJU/MeMOTR/blob/main/outputs/memotr_mot17/train/config.yaml).

Now, as this [commit](https://github.com/MCG-NJU/MeMOTR/commit/619574f0082d5e2fed9270bcf1f0401dfbf889e2), I have already uploaded the config file. Thanks a lot for your reminder.

**Comment by sawhney-medha** [comment]
Thank you so much!!

**Comment by HELLORPG** [comment]
I think you may be conducting experiments on MOT17, and there are some things that I need to explain to you. I did not reply earlier because I was too busy in the past several days. Sorry about that.

First of all, due to the merging of several versions of code, there was a problem in passing parameters during data processing, which I have corrected in this [commit](https://github.com/MCG-NJU/MeMOTR/commit/ac32339bf22f9e0ec51f41031013232eaa8c1d14).

Moreover, there are some differences in the configuration on MOT17 (compared with DanceTrack/SportsMOT):
1. The optimizer setting:
    ```yaml
    LR_POINTS: 2.0e-5    # 1.0e-5 on DanceTrack
    WEIGHT_DECAY: 0.0001    # 0.0005 on DanceTrack
    ```
    I set the `lr_points` and `weight_decay` to 2.0e-5 and 0.0001 because of the stable convergence. This setting requires more training epochs, e.g. 18 (in our paper) to 20 (in our open-source code) on DanceTrack. However, as I discussed in #11, training on MOT17 is a delicate balance, so I haven't tried this new training configuration yet to see if it can achieve the desired results.
2. The image size during augmentation:
    ```yaml
    COCO_SIZE: True    # False on DanceTrack
    ```
    This setting comes from [MOTR](https://github.com/megvii-research/MOTR). I don't quite understand the reason for this, but it does achieve better results.
3. Whether the bounding boxes can exceed the image size:
    ```yaml
    OVERFLOW_BBOX: True    # False on DanceTrack
    ```
    Referring to some previous work, this approach can achieve better detection performance, but it will have a loss on the AssA metric. This is a difficult balance to choose.

BTW, it is also very common in MOT17 that newborn objects are suppressed by tracked targets, resulting in detection failure. Several recent research (CO-MOT/MOTRv3) focus on this. We hope we can resolve this issue in the future.
